src/css/global.css

```diff
@@ -115,16 +111,12 @@
   outline: none;
   border: none;
   font-size: 16px;
-  margin-bottom: var(--spacingSm);
+  /* margin-bottom: var(--spacingSm); */
```

Comment:
```suggestion
```

Reply:
Removed in 6e56c2be78e42db03e6abdbd5075c3d17e03b888